packages/alba/alba.0.4.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name: "alba"
```

Comment:
```suggestion
```
This is redundant, will be overriden by opam using the name from the opam file path